openbb_terminal/common/behavioural_analysis/twitter_view.py

```diff
@@ -102,84 +106,89 @@ def display_sentiment(
 
     df_tweets = twitter_model.get_sentiment(symbol, n_tweets, n_days_past)
 
-    if not df_tweets.empty:
-
-        ax1, ax2, ax3 = None, None, None
-
-        if compare:
-            # This plot has 3 axes
-            if external_axes is None:
-                _, axes = plt.subplots(
-                    3, 1, sharex=False, figsize=plot_autoscale(), dpi=cfg_plot.PLOT_DPI
-                )
-                ax1, ax2, ax3 = axes
-            elif is_valid_axes_count(external_axes, 3):
-                (ax1, ax2, ax3) = external_axes
-            else:
-                return
+    # given object is a dataframe and is empty
+    if isinstance(df_tweets, pd.DataFrame) and df_tweets.empty:
+        return
+    # given object is not a dataframe and is None
+    if not isinstance(df_tweets, pd.DataFrame) and not df_tweets:
+        return
```

Comment:
Why not do:
if not df_tweets:
return pd.DataFrame()
if df_tweets.empty:
return pd.DataFrame()
(sorry its terribly formatted)